src/frontend/src/lib/components/transactions/Transaction.svelte

```diff
@@ -18,6 +18,7 @@
 	export let timestamp: number | undefined;
 	export let styleClass: string | undefined = undefined;
 	export let token: Token;
+	export let hideTokenLogo = false;
```

Comment:
Another option is something like as following

```
export let badge: "token" | "logo" = "logo";
```

Reply:
way better!!! thank you! changed to iconType! have a look!